R/helpers_knitr_NextBest.R

```diff
@@ -1,5 +1,3 @@
-# Integration with knitr ----
-#'
 #' @description `r lifecycle::badge("experimental")`
```

Comment:
this part would be important to only include once in one of the R files. otherwise we would get roxygen2 to "overwrite" this multiple times, and for further maintenance it would be unclear which file to edit (or all of them need to be consistently updated, which does not make sense)

Reply:
Yes.  For `helpers_knitr_NextBest.R`.  Do you want me to do it for the other `helpers_knitr_xxxx.R` files?

Reply:
Yes please check that there is no duplicate documentation, otherwise please remove it and only keep a single instance of the documentation

Reply:
Done.